src/lib/storage/index/adaptive_radix_tree/adaptive_radix_tree_index.hpp

```diff
@@ -37,6 +37,12 @@ class AdaptiveRadixTreeIndex : public BaseIndex {
   friend class AdaptiveRadixTreeIndexTest_BinaryComparableFromChunkOffset_Test;
 
  public:
+  /**
+   * Predicts the memory consumption in bytes of creating this index.
+   * See BaseIndex::predict_memory_consumption()
+   */
+  static size_t predict_memory_consumption(ChunkOffset row_count, ChunkOffset value_count, uint32_t value_bytes);
```

Comment:
for Tables we call this "estimate_memory_consumption()", I think. Let's keep the names consistent. Same for "size_t _memory_consumption() const final;", that could be "_estimate_memory_consumption_(impl)()" or something

Reply:
renamed to `estimate_memory_consumption()` in BaseIndex and its derived classes.